<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/*-----QWidget-----*/   
QWidget
{
	/*background-color: rgb(29, 29, 30);*/
	color: rgb(31, 47, 18);
	selection-background-color: #b78620;
	selection-color: #000;

}




/*-----QLabel-----*/
/*
QLabel
{
	background-color: transparent;
	color: red;

}
*/


/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
	color: #ffffff;
	min-width: 80px;
	border-style: solid;
	border-width: 1px;
	border-radius: 6px;
	border-color: #051a39;
	padding: 5px;

}


QPushButton::flat
{
	background-color: transparent;
	border: none;
	color: #fff;

}


QPushButton::disabled
{
	background-color: #404040;
	color: #656565;
	border-color:  rgb(175, 168, 167)    /*#051a39;*/

}


QPushButton::hover
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;

}


QPushButton::pressed
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(74, 74, 74, 255),stop:1 rgba(49, 49, 49, 255));
	border: 1px solid #b78620;

}


QPushButton::checked
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(74, 74, 74, 255),stop:1 rgba(49, 49, 49, 255));
	border: 1px solid #222;

}


/*-----QPlainTExtEdit-----*/
QPlainTextEdit
{
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 8px;
	padding: 3px;
	padding-left: 5px;

}

/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #fff;

}

/*-----QLineEdit-----*/
QLineEdit
{
	
	selection-background-color: darkgray;
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 4px;
	padding: 3px;
	padding-left: 5px;

}


QLineEdit:read-only
{
	background-color: rgb(47, 49, 49);
	color : #eee;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
    color: lightgray;
	border: none;

}

QCheckBox::indicator:unchecked:hover
{
	border: 1px solid #b78620; 
	border-radius: 5px;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}

/*-----QComboBox-----*/
QComboBox
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
    border: 1px solid #000;
    padding-left: 6px;
    color: #ffffff;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QComboBox:on
{
    background-color: #b78620;
	color: #000;

}


QComboBox QAbstractItemView
{
    background-color: #383838;
    color: #ffffff;
    border: 1px solid black;
    selection-background-color: #b78620;
    outline: 0;

}


QComboBox::drop-down
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid; 

}


QComboBox::down-arrow
{
    image: url(://arrow-down.png);
    width: 8px;
    height: 8px;
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="hastTableView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>231</width>
      <height>331</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HashTable</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="removeEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="seatchResultEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="hastTextEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>261</width>
      <height>321</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HashTable values</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="randHashButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>400</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Hash Value</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
